compiler/cstc_repl/tests/repl_session.cpp

```
@@ -0,0 +1,372 @@
+    test_expression_statements_are_not_persisted();
+    test_item_definitions_persist_without_running_previous_turns();
+    test_relative_imports_resolve_from_session_root();
+    test_supported_result_types_are_rendered();
+    test_unsupported_result_types_report_a_notice();
+    test_commands_and_reset_clear_persisted_state();
+    test_quit_command_requests_exit();
+    test_dot_commands_are_rejected();
+    test_startup_text_mentions_help_and_quit();
+    test_runtime_errors_do_not_commit_new_state();
+    test_missing_linker_reports_process_start_failure();
+    test_custom_linker_path_with_spaces_is_supported();
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)